awx/settings/development.py

```diff
@@ -21,6 +21,8 @@
 # Load default settings.
 from .defaults import *  # NOQA
 
+# How much capacity controlling a task costs a node
+AWX_CONTROL_NODE_TASK_IMPACT = 5
```

Comment:
What's the difference between this variable and `AWX_CONTROL_PLANE_TASK_IMPACT` in `defaults.py`?

Reply:
TBH one tests was failing on not picking up the setting in `defaults.py`. I can revert this part and try and get to the bottom of that

Reply:
finally realized these were just two different words :D. fixed/deduplicated now